README.md

```diff
@@ -1,69 +1,99 @@
+- [`ar4_gazebo_bringup`](./ar4_gazebo): Launch a Gazebo Sim simulation of AR4.
+- [`ar4_isaac_bringup`](./ar4_gazebo): **Not yet available.** Launch an Isaac Sim simulation of AR4.
+- [`ar4_mujoco_bringup`](./ar4_gazebo): **Not yet available.** Launch a Mujoco-based simulation of AR4.
+- [`ar4_realbot_bringup`](./ar4_gazebo): Launch drivers and control software for the real AR4 robot.
 - [`ar4_isaac`](./ar4_isaac): Isaac simulation for the ar4 arm.
 
-## Installation
 
-### Prerequisites
+```mermaid
+graph TD
+      A[**ar4_moveit_config** <br> main.launch.py]
+      B[**ar4_common** <br> main.launch.py]
+      C[**ar4_description** <br> main.launch.py]
+      D[**ar4_gazebo_sim** <br> main.launch.py]
+      E[**ar4_gazebo_bringup** <br> main.launch.py]
+      F[**ar4_hardware_interface** <br> main.launch.py]
+      G[**ar4_realbot_bringup** <br> main.launch.py]
 
-It is a requirement to have `docker engine` with the `docker compose plugin` already installed in the host machine.
+      A --> F
+      C --> F
 
-See: [Docker Installation Guide](https://docs.docker.com/engine/install/ubuntu/)
+      H[**ar4_isaac_bringup** <br> main.launch.py <br> ...not yet available...]
+      I[**...** <br> main.launch.py]
 
-### Running the dev container
+      B --> H
+      I --> H
 
-Build and run the container for the use case you are interested in
+      J[**ar4_mujoco_bringup** <br> main.launch.py <br> ...not yet available...]
+      K[**...** <br> main.launch.py]
 
-#### Gazebo
-```bash
-./docker/run.sh -s ar4_gazebo
-```
+      B --> J
+      K --> J
 
-#### Hardware
 
-* [Hardware Interface Instructions](ar4_hardware_interface/README.md)
+      A --> B
 
-## Build the packages
+      C --> D
+      C --> A
 
-Build the packages
+      B --> E
+      D --> E
 
-```
-colcon build
+      B --> G
+      F --> G
 ```
 
-Source the built packages
+## Installation
 
-```
-source install/setup.bash
-```
+### Prerequisites
 
----
+It is a requirement to have `docker engine` with the `docker compose plugin` already installed in the host machine.
 
-### Launch
+See: [Docker Installation Guide](https://docs.docker.com/engine/install/ubuntu/)
 
-After building and sourcing the packages, run the specific launch file for your use case:
+### Running the dev container
+
+Build and run the container for the use case you are interested in
 
 #### Gazebo
 
+Launch the docker container with the following command:
+
+```bash
+./docker/run.sh -s ar4_gazebo
 ```
-ros2 launch ar4_gazebo moveit.launch.py
+
+Then build and launch Gazebo bringup launch file:
+
+```bash
+colcon build --symlink-install --packages-up-to ar4_gazebo_bringup \
+ && source install/setup.bash \
+ && ros2 launch ar4_gazebo_bringup main.launch.py
 ```
 
 ![Ar4 Gazebo](docs/ar4.png)
 
----
+
+#### Hardware
+
+* [Hardware Interface Instructions](ar4_hardware_interface/README.md)
+
 
 ### Controlling the AR4 with MoveIt
 
```

Comment:
I think it'll be good to reference MuJoCo in this `README`, similar to what we do with Isaac.

Reply:
Fixed.